server/src/main/java/org/tbbtalent/server/model/db/CandidateVisaJobCheckBase.java

```diff
@@ -30,8 +30,19 @@ public class CandidateVisaJobCheckBase extends AbstractDomainObject<Long> {
     @JoinColumn(name = "candidate_visa_check_id")
     private CandidateVisaCheck candidateVisaCheck;
 
+    /**
+     * Associated job opportunity
+     */
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "job_opp_id")
+    SalesforceJobOpp jobOpp;
+
+    /**
+     * No longer populating these two fields (name & sfJobLink) we used these prior to adding the SF job opps onto the TC,
+     * and when Australia was the only country doing their visa checks on the TC. 
```

Comment:
If you haven't already, can you create an issue to remove these old fields when they are no longer used anywhere?